### DISCUSSION TITLE
Working in New update for track users in List chat automaticly

### DISCUSSION THREAD

**Discussion by Spipous200** [rizwansoaib/whatsapp-monitor#91] [Ideas]
Hii now iam working in updating extensión for track users automaticly from your chatlist elso  Will be soon for Contact if you are interssing comment me here and i Will share the update as soon as possible

**Comment by rizwansoaib** [comment]
Share your ideas people will see it...

**Reply by Spipous200** [reply]
Is working with wapi store to Fetch all chatids un store then creat presance for them automaticly for all chats

**Comment by Spipous200** [comment]
chat_list =  window.WAPI.getAllChatIds();
cnt =  chat_list.forEach(function(cnt, index) {
          setTimeout(function(){
          console.log(cnt)
            Store.Chat.get(cnt).presence.subscribe()
          }, 10 * (index + 1));



This is Snipet Code you can try It from console after excursión in consomé u Will see the tracking start for all your chat in liste

**Comment by rizwansoaib** [comment]
Can we do it for non Saved contact . @Spipous200 

We can add in next update this snippet codes.





On Wed 14 Jul, 2021, 4:11 AM Spipous200, ***@***.***> wrote:

> chat_list = window.WAPI.getAllChatIds();
> cnt = chat_list.forEach(function(cnt, index) {
> setTimeout(function(){
> console.log(cnt)
> Store.Chat.get(cnt).presence.subscribe()
> }, 10 * (index + 1));
>
> This is Snipet Code you can try It from console after excursión in consomé
> u Will see the tracking start for all your chat in liste
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/91#discussioncomment-1001249>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZDEUPRNCQLW4LARXFTTXS6KFANCNFSM5AHPB7PQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by Spipous200** [reply]
This work for not saved Contact but we cant creat presance for not saved Contact if is not in chat list
